airflow/models/taskinstance.py

```diff
@@ -139,6 +139,12 @@
     from airflow.models.operator import Operator
     from airflow.utils.task_group import MappedTaskGroup, TaskGroup
 
+    # This is a workaround because mypy doesn't work with hybrid_property
+    # TODO: remove this hack and move hybrid_property back to main import block
```

Comment:
```suggestion
    # TODO: remove this hack and move hybrid_property back to main import block
    # See https://github.com/python/mypy/issues/4430
```

Reply:
I suggest to add link to the specific issue here. It will be easier for anyone in the future to see it the issue is solved.

Reply:
(that is if I found the right issue of course)